Add container files

IceflowRE opened the pull request:
There was/is no documentation on how to run the asset library. I hope that the container files will add more to this and provide an easy way to spin up the asset-library.

Thanks to @Piralein for helping me!

IceflowRE committed:
Add container files

Calinou commented on container/Dockerfile:
```diff
@@ -0,0 +1,14 @@
+FROM php:7.4-apache
+
+RUN apt update && apt install -y git zip \
```
Docker recommends using apt-get over apt for automated tasks, as its API is less likely to break over time:

```suggestion
RUN apt-get update && apt-get install -y git zip \
```

Calinou commented on container/Dockerfile:
```diff
@@ -0,0 +1,14 @@
+COPY . /var/www/html/asset-library
+
+WORKDIR /var/www/html/asset-library
+RUN composer install
+RUN a2enmod rewrite
+RUN chown -R www-data:www-data .
```
Should this be condensed to a single `RUN` command to avoid creating new layers (therefore making image builds faster and smaller)?

```suggestion
RUN composer install \
    && a2enmod rewrite \
    && chown -R www-data:www-data .
```

IceflowRE committed:
Improve container setup

IceflowRE commented:
Also moved the `docker-compose.yml` to `docker-compose.example.yml` and adjusted the readme to create the docker compose file yourself. This makes it easier to pull the repo later on, if you adjust the docker compose file.
